omniduct/databases/base.py

```diff
@@ -353,7 +357,8 @@ def _get_formatter(self, formatter, cursor, **kwargs):
         if not (inspect.isclass(formatter) and issubclass(formatter, _cursor_formatters.CursorFormatter)):
             assert formatter in self.CURSOR_FORMATTERS, "Invalid format '{}'. Choose from: {}".format(formatter, ','.join(self.CURSOR_FORMATTERS.keys()))
             formatter = self.CURSOR_FORMATTERS[formatter]
-        return formatter(cursor, **kwargs)
+        format_opts = dict(list(self._default_format_opts.items()) + list(kwargs.items()))
```

Comment:
Nothing major, but I tend to prefer chaining iterators rather than explicitly converting things to lists: `dict(itertools.chain(self._default_format_opts.items(), kwargs.items()))`